cradle-core/src/main/java/com/exactpro/cradle/testevents/BatchedStoredTestEvent.java

```diff
@@ -66,8 +66,21 @@ public BatchedStoredTestEvent(TestEventSingle event, TestEventBatch batch, PageI
 		this.batch = batch;
 		this.pageId = pageId;
 	}
-	
-	
+
+	protected BatchedStoredTestEvent(StoredTestEventId id, String name, String type, StoredTestEventId parentId,
```

Comment:
Use this constructor in constructor above

Reply:
There is an additional logic in first constructor